src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vm/lib/models/parameters_link.py

```diff
@@ -14,8 +15,7 @@
 
 
 class ParametersLink(Model):
-    """
-    Entity representing the reference to the deployment paramaters.
+    """Entity representing the reference to the deployment paramaters.
```

Comment:
nit (and probably not related to your change) but there's a typo with 'paramaters'